lib/torrent.js

```diff
@@ -252,6 +254,14 @@ class Torrent extends EventEmitter {
   _processParsedTorrent (parsedTorrent) {
     this._debugId = parsedTorrent.infoHash.toString('hex').substring(0, 7)
 
+    if (typeof this.private !== 'undefined') {
+      // `private` option overrides default, only if it's defined
+      parsedTorrent.private = this.private
+    } else {
+      // use default value
+      this.private = !!parsedTorrent.private
```

Comment:
The Object.assign below will assign `this.private` to `parsedTorrent.private`, so the else block isn't necessary? Or maybe there's a bug here?

Reply:
Good one! Well, if the parsed torrent is public then `parsedTorrent.private` would be `undefined`. Thus, the variable `this.private` would not be updated in the Object.assign. This line basically forces this update.

Reply:
It would be falsy either way, but I guess having it defined is good? seems fine

Reply:
Ok, done. I think your change even improves readability. Thanks :)